src/pages/studies/StudyTableContainer.tsx

```diff
@@ -24,26 +30,29 @@ const connector = connect(mapState, mapDispatch);
 
 type PropsFromRedux = ConnectedProps<typeof connector>;
 export type PaginationType = {
-  pagination: {
-    current: number;
-    pageSize: number;
-    total: number;
-    onChange: (page: number) => void;
-  };
+  pagination: TablePaginationConfig;
 };
 
 type Props = StudiesResults & PropsFromRedux & PaginationType;
 
 const StudyTable: FC<Props> = (props) => {
   const { loading, pagination } = props;
+  const { current: currentPage, total: itemTotal = 0, pageSize: itemPerPage = 10 } = pagination;
+
   if (loading) {
     return null;
   }
 
   const tableData = generateTableData(props);
+  const pageRange = `${currentPage}-${itemTotal > itemPerPage ? itemPerPage : itemTotal}`;
```

Comment:
Personally, I would create a util function since it's not obvious for me at first sight what it does. However, I cannot objectively object to this line of code.

Reply:
I don't think a function will make it clearly, maybe variable name